.github/workflows/wordlist.txt

```diff
@@ -17,6 +17,9 @@ TPSF
 accelerometer
 timeDelayWidths
 dt
+Gowerlabs
+gowerlabs
```

Comment:
I think this one can be removed, right? Not sure if we need case sensitivity? Otherwise, the list will grow very long.

Reply:
I’m not sure. But I think we should tidy this list up anyway. Maybe make it sorted alphabetically, it’s a mess now. But let’s do that in a seperate PR